packit_service/constants.py

```diff
@@ -6,6 +6,7 @@
 
 CONFIG_FILE_NAME = "packit-service.yaml"
 
+PACKIT_ACCOUNT_CHECK = "packit/valid-fas-account"
```

Comment:
```suggestion
PACKIT_ACCOUNT_CHECK = "packit/authorized-to-build"
```

FAS is Fedora and we want less fedora :)